src/DDM.jl

```diff
@@ -1,81 +1,182 @@
 """
-    DDM{T<:Real} <: SSM2D
+    DDM{T<:Real} <: AbstractDDM
 
-Model object for the standard Drift Diffusion Model.
+    Model object for the Ratcliff (Full) Diffusion Decision Model.
 
 # Parameters
-- `ν`: drift rate. Average slope of the information accumulation process. The drift gives information about the speed and direction of the accumulation of information. Typical range: -5 < ν < 5
-- `α`: boundary threshold separation. The amount of information that is considered for a decision. Typical range: 0.5 < α < 2
-- `τ`: non-decision time. The duration for a non-decisional processes (encoding and response execution). Typical range: 0.1 < τ < 0.5 
-- `z`: starting point. Indicator of an an initial bias towards a decision. The z parameter is relative to a (i.e. it ranges from 0 to 1).
+    - `ν`: drift rate. Average slope of the information accumulation process. The drift gives information about the speed and direction of the accumulation of information. Typical range: -5 < ν < 5
```

Comment:
The parameter descriptions need to be aligned with the # Parameters header. Otherwise, they will not render.